plugins/pencilblue/templates/admin/login.html

```diff
@@ -38,7 +38,7 @@
                                 </div>
                                 <input type="button" class="btn btn-lg btn-primary btn-block" value="^loc_LOGIN^" onclick="login()">
                                 <div class="forgot_password">
-                                    <a href="javascript:forgotPassword()">^loc_FORGOT_PASSWORD^</a>
+                                    <a href="javascript:forgotPassword()">I forgot my password</a>
```

Comment:
So we can blame this on @brianhyder.

![dissapointed](http://i.imgur.com/26k9Td7.gif)